prov/verbs/src/verbs_cq.c

```diff
@@ -325,7 +325,14 @@ static ssize_t fi_ibv_cq_read(struct fid_cq *cq_fid, void *buf, size_t count)
 			break;
 
 		/* Insert error entry into wcq */
-		if (wc.status) {
+		if (OFI_UNLIKELY(wc.status)) {
+			if (wc.status == IBV_WC_WR_FLUSH_ERR) {
+				/* Handle case when remote side destroys
+				 * the connection, but local side isn't aware
+				 * about that yet */
+				i--;
```

Comment:
Can we add a debug log for ignoring IBV_WC_WR_FLUSH_ERR`?

Reply:
Sure